src/test/java/com/kodilla/ecommercee/mapper/GroupMapperTestSuite.java

```diff
@@ -0,0 +1,66 @@
+package com.kodilla.ecommercee.mapper;
+import com.kodilla.ecommercee.domain.Group;
+import com.kodilla.ecommercee.domain.dto.GroupDto;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import java.util.ArrayList;
+import java.util.List;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class GroupMapperTestSuite {
+
+    @Autowired
+    private GroupMapper groupMapper;
+
+    @Test
+    public void testMapToGroup() {
+        //Given
+        GroupDto groupDto = new GroupDto(1L, "Test group");
```

Comment:
Hmm co sądzisz o wyznaczeniu dwóch metod zwracających ci obiekty klasy Group i GroupDto , abyś nie musiał tworzyć stale ich obiektów a jedynie wywołać metodę ? 

Metoda dla GroupDto przykładowo wyglądała by tak :
public GroupDto getGroupDtoContent() {
return new GroupDto(1L, "Test group");
}

I wtedy linijka 24 z tego: GroupDto groupDto = new GroupDto(1L, "Test group");
wyglądałą by tak : GroupDto groupDto = getGroupDtoContent();
Daj znać co sądzisz. :D

Reply:
W tym wypadku nie ma za dużo danych, ale przyszłościowo to jest całkiem niezłe wyjście, po prostu robiłem to na przykładzie UserMapperTestSuite, może wypadałoby ujednolicić testy :)